docs/lit/examples/7-osem.jl

```diff
@@ -0,0 +1,204 @@
+#---------------------------------------------------------
+# # [SPECTrecon OS-EM](@id 7-osem)
+#---------------------------------------------------------
+
+# This page illustrates OS-EM reconstruction with the Julia package
+# [`SPECTrecon`](https://github.com/JeffFessler/SPECTrecon.jl).
+
+# ### Setup
+
+# Packages needed here.
+
+using SPECTrecon # need to fix the "project idx" bug first!
+using MIRTjim: jim, prompt
+using Plots: scatter, plot!, default; default(markerstrokecolor=:auto)
+using LinearMapsAA: LinearMapAA, LinearMapAO
+using LinearAlgebra: mul!
+using Distributions: Poisson
+# The following line is helpful when running this example.jl file as a script;
+# this way it will prompt user to hit a key after each figure is displayed.
+
+isinteractive() ? jim(:prompt, true) : prompt(:draw);
+
+# ### Overview
+
+# Ordered-subset expectation-maximization (OS-EM)
+# is a commonly used algorithm for performing SPECT image reconstruction.
+
+# ### Simulation data
+
+nx,ny,nz = 64,64,50
+T = Float32
+xtrue = zeros(T, nx,ny,nz)
+xtrue[(1nx÷4):(2nx÷3), 1ny÷5:(3ny÷5), 2nz÷6:(3nz÷6)] .= 1
+xtrue[(2nx÷5):(3nx÷5), 1ny÷5:(2ny÷5), 4nz÷6:(5nz÷6)] .= 2
+
+average(x) = sum(x) / length(x)
+function mid3(x::AbstractArray{T,3}) where {T}
+    (nx,ny,nz) = size(x)
+    xy = x[:,:,ceil(Int, nz÷2)]
+    xz = x[:,ceil(Int,end/2),:]
+    zy = x[ceil(Int, nx÷2),:,:]'
+    return [xy xz; zy fill(average(xy), nz, nz)]
+end
+jim(mid3(xtrue), "Middle slices of xtrue")
+
+
+# ### PSF
+
+# Create a synthetic depth-dependent PSF for a single view
+px = 11
+psf1 = psf_gauss( ; ny, px)
+jim(psf1, "PSF for each of $ny planes")
```

Comment:
```suggestion
jim(psf1, "PSF for each of $ny planes"; ratio=1)
```
This is a new option that looks better here :)